topics/microbiome/tutorials/multivariable-association/workflows/Workflow-constructed-from-history-'Determining-multivariable-association-between-various-meta’omic-features-using-MaAslin2.ga

```diff
@@ -0,0 +1,129 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "comments": [],
+    "format-version": "0.1",
+    "name": "Workflow constructed from history 'Determining multivariable association between various meta\u2019omic features using MaAslin2",
```

Comment:
```suggestion
    "name": "Determining multivariable association between various meta-omic features using MaAslin2",
```
This way?

Reply:
I kept the omitted part just to differentiate a between the title of  workflow file and tutorial file.  If it doesn't cause any issue then I believe we should not change it. What are your thoughts on this ? @bgruening

Reply:
I don't see any reason why a workflow should be named 'construceted from history'

Reply:
Can you please remove the two workflows, starting with Workflow constructed from history ...` and only keep the main one with the correct title ?

Reply:
https://github.com/galaxyproject/training-material/pull/5173/commits/b10e57a0af540e0a4f5d503c06bcb472a99dd9f9 ,  https://github.com/galaxyproject/training-material/pull/5173/commits/74a3f39a86ce7b1ea7e47a27aecc57641167c5a7,  https://github.com/galaxyproject/training-material/pull/5173/commits/0e828c135f0cd8448f72c5ddac66e29bf714f32a